src/rabbit_misc.erl

```diff
@@ -71,6 +71,7 @@
 -export([store_proc_name/1, store_proc_name/2]).
 -export([moving_average/4]).
 -export([get_env/3]).
+-export([get_channel_operation_timeout/0]).
```

Comment:
@hairyhum yes it could. `channel_operation_timeout` is going to be used in various places, and had a brief chat with @michaelklishin about making it accessible from a common module. & chose to use `rabbit_misc`.